### PR TITLE
adafruit-ampy: Init at 1.0.5

### PR DESCRIPTION
**Pull request by ivanovsaleksejs**
###### Motivation for this change

Adafruit MicroPython Tool (ampy) - Utility to interact with a MicroPython board over a serial connection.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/python-dotenv/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, buildPythonPackage, pythonPackages }:
```
Comment (FRidh) [inline pkgs/development/python-modules/python-dotenv/default.nix]: Don't pass in `pythonPackages`. Instead pass in the individual dependencies.

File: pkgs/development/python-modules/python-dotenv/default.nix [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+buildPythonPackage rec {
+  pname = "python-dotenv";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
```
Comment (FRidh) [inline pkgs/development/python-modules/python-dotenv/default.nix]: Is it possible to fetch from Pypi?

File: pkgs/development/python-modules/python-dotenv/default.nix [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  checkInputs = with pythonPackages; [
+    ipython
+    pytest
+    sh
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/theskumar/python-dotenv;
+    description = "Reads the key,value pair from .env file and adds them to environment variable";
+    platforms = platforms.all;
```
Comment (FRidh) [inline pkgs/development/python-modules/python-dotenv/default.nix]: Don't set platforms, it's set already.

File: pkgs/development/python-modules/python-dotenv/default.nix [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    license = licenses.bsd3;
+    maintainers = [];
```
Comment (FRidh) [inline pkgs/development/python-modules/python-dotenv/default.nix]: are you going to maintain this package?

File: pkgs/development/tools/adafruit-ampy/default.nix [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  ];
+
+  # No tests
+  doCheck = false;
+
+  meta = {
+    description = "Utility to interact with a MicroPython board over a serial connection";
+    homepage = https://github.com/adafruit/ampy;
+    license = stdenv.lib.licenses.mit;
+    maintainers = [];
```
Comment (FRidh) [inline pkgs/development/tools/adafruit-ampy/default.nix]: are you going to maintain this package?

File: pkgs/development/tools/adafruit-ampy/default.nix [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    platforms = stdenv.lib.platforms.unix;
```
Comment (FRidh) [inline pkgs/development/tools/adafruit-ampy/default.nix]: is darwin not supported?

**Review by FRidh — CHANGES_REQUESTED** [review]

**Comment by timokau** [comment]
https://github.com/NixOS/nixpkgs/pull/61495

**Pull request closed without merging**